src/iris/api.py

```diff
@@ -700,6 +710,28 @@ def process_resource(self, req, resp, resource, params):  # pragma: no cover
         if resource.allow_read_no_auth and method == 'GET':
             return
 
+        # Ignore HMAC requirements for custom webhooks
+        if len(req.env['PATH_INFO'].split("/")) > 2 and req.env['PATH_INFO'].split("/")[2] == 'webhook':
+            qs = parse_qs(req.env['QUERY_STRING'])
+
+            app_name = qs['application'][0]
```

Comment:
Instead of using parse_qs/etc and manually decoding the query string, please use falcon's built-in helpers if possible.

```
app = req.get_param('application', required=True)
api_key = req.get_param('key', required=True)
```